auth0_flutter_platform_interface/test/method_channel_auth0_flutter_web_auth_test.dart

```diff
@@ -169,6 +170,47 @@ void main() {
       expect(result.refreshToken, isNull);
     });
 
+    test('correctly includes safariViewController options when specified',
+        () async {
+      when(mocked.methodCallHandler(any))
+          .thenAnswer((final _) async => MethodCallHandler.loginResult);
+
+      await MethodChannelAuth0FlutterWebAuth().login(
+          WebAuthRequest<WebAuthLoginOptions>(
+              account: const Account('test-domain', 'test-clientId'),
+              userAgent: UserAgent(name: 'test-name', version: 'test-version'),
+              options: WebAuthLoginOptions(
+                  safariViewController: const SafariViewController(
+                      presentationStyle:
+                          SafariViewControllerPresentationStyle.formSheet))));
+
+      final verificationResult =
+          verify(mocked.methodCallHandler(captureAny)).captured.single;
+
+      expect(
+          verificationResult.arguments['safariViewController']
+              ['presentationStyle'],
+          2);
```

Comment:
Should we use `SafariViewControllerPresentationStyle.formSheet.value` here?

Reply:
Yes!